lib/ansible/plugins/strategy/free.py

```diff
@@ -162,7 +162,12 @@ def run(self, iterator, play_context):
                             # we don't care if it just shows the raw name
                             display.debug("templating failed for some reason", host=host_name)
 
-                        run_once = templar.template(task.run_once) or action and getattr(action, 'BYPASS_HOST_LOOP', False)
+                        templated_task_run_once = templar.template(task.run_once)
+                        if templated_task_run_once is not None:
+                            task_run_once = task.get_validated_value('run_once', task.fattributes.get('run_once'), templated_task_run_once, templar)
```

Comment:
get_validated_value already templates, you do not need to do both

Reply:
but I found there is not use templates on `isa boll`

Reply:
@bcoca Hi, here's a question for waitting to reply. please~